bob.yaml

```diff
@@ -18,6 +18,7 @@ node:
 
 chain:
   url: ws://127.0.0.1:8545
+  contractsetup: validate
```

Comment:
What we just talked about:  
Either make it an `enum` and then parse the enum value in the `SetConfig` function (maybe `viper` can do enums?) 
or document and validate the possible string values, so that the user does not enter bogus.

Reply:
as far as i could find out, viper does not support enums. i documented valid options in config.go. additionally i revised the setupContracts method to ensure that an expressive error is thrown if the option setupcontract is not specified correctly.

Reply:
after offline discussion, now using separate type contractSetupOption and validating configuration during startup in config.go.